### PR TITLE
CDK: migration steps for partners

### PR DESCRIPTION
**Pull request by kmurphypolygon**
https://polygon.atlassian.net/browse/DOCS-110

### CODE REVIEW

**Comment by Sharonbc01** [comment]
Hi @kmurphypolygon please do not publish yet. I have asked product @gsilva5 to provide some verbiage for the first section on how CDK owners / IP partners contact Polygon to schedule required upgrade transactions.

File: docs/cdk/how-to/migrate/7-8.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+This document shows Polygon partners how to migrate a CDK stack from fork 7 to fork 8.
+
+!!! tip
+    This process is done collaboratively with Polygon the CDK team.
+
+## Process to upgrade from fork 7 to 8
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
## Process to upgrade from fork 7 to 8 for isolated CDK chains
```

**Comment by Sharonbc01** [comment]
@kmurphypolygon  I added a note to indicate this upgrade path is for isolated CDK chains only.

Please can you remove steps 1 to 8. Gabe advised we will create better verbiage for IP Partner from Fork ID 10 onwards when we really need to have this process in place. 

Thus starting the body of the article from Verify all batches works for isolated CDK chains. Thanks

File: docs/cdk/how-to/migrate/7-8.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    !!! warning
+        The current batch verification procedure relies on stopping the sequencer.
+
+5. After successful batch verification, CDK operators inform Polygon that their CDK is ready for the upgrade.
+
+6. Polygon executes the scheduled transaction on the timelock, making the upgrade effective.
+
+7. Polygon informs the CDK operators of the transaction confirmation. (It is also possible to check Etherscan for confirmation of the same.)
+
+8. CDK operators update the software and restart all components. Follow the [update software](#update-software) instructions below.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Verify all batches
+
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+This document shows Polygon partners how to migrate a CDK stack from fork 7 to fork 8.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
This document shows Polygon partners how to migrate an isolated  CDK stack from fork 7 to fork 8.
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+!!! tip
+    This process is done collaboratively with Polygon the CDK team.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/7-8.md]: please delete this sentence

File: docs/cdk/how-to/migrate/7-8.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+    8. Wait until the proof is settled on-chain: 
+
+        `SELECT batch_num FROM state.verified_batch ORDER BY batch_num DESC LIMIT 1;` → X
+
+3. Prepare (**do not apply**) new versions according to the version matrix.
+
+4. Stop all services (node, prover/executor, bridge).
+
+5. Ask Polygon to send the L1 tx, and wait until confirmation.
```
Comment (Sharonbc01) [inline docs/cdk/how-to/migrate/7-8.md]: Remove this sentence

**Review by Sharonbc01 — CHANGES_REQUESTED** [review]
@kmurphypolygon  as we had to move to Fork ID 9 over the weekend due to zkEVM outage this should probably be reflected. best to replace 8 with 9 through out

File: docs/cdk/how-to/migrate/7-8.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
+This document shows Polygon partners how to migrate an isolated  CDK stack from fork 7 to fork 8.
+
+!!! tip
+    This process is done collaboratively with Polygon the CDK team.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+    8. Wait until the proof is settled on-chain: 
+
+        `SELECT batch_num FROM state.verified_batch ORDER BY batch_num DESC LIMIT 1;` → X
+
+3. Prepare (**do not apply**) new versions according to the version matrix.
+
+4. Stop all services (node, prover/executor, bridge).
+
+5. Ask Polygon to send the L1 tx, and wait until confirmation.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
+This document shows Polygon partners how to migrate an isolated  CDK stack from fork 7 to fork 8.
+
+
+## Process to upgrade from fork 7 to 8 for isolated CDK chains
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
## Process to upgrade forks for isolated CDK chains
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
+This document shows Polygon partners how to migrate an isolated  CDK stack from fork 7 to fork 8.
+
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 11]
```diff
@@ -0,0 +1,73 @@
+This document shows Polygon partners how to migrate an isolated  CDK stack from fork 7 to fork 8.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
This document shows Polygon partners how to migrate an isolated CDK stack.
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
+This document shows Polygon partners how to migrate an isolated CDK stack.
+
+## Process to upgrade forks for isolated CDK chains
+
+In order to avoid reorgs and other undesirable scenarios, all L2 transactions must be verified before upgrading a fork. Verification means that all batches are closed, sequenced, and verified on L1.
+
+Follow the steps to verify all batches for upgrading from fork 7 to 8.
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
Follow the steps to verify all batches for upgrading.
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+3. Prepare (**do not apply**) new versions according to the version matrix.
+
+4. Stop all services (node, prover/executor, bridge).
+
+
+## Update software
+
+1. Start synchronizer's new version.
+
+2. Wait until synchornizer receives a fork id 8 event (check table `state.fork_id`).
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
2. Wait until synchornizer receives a fork id event (check table `state.fork_id`).
```

File: mkdocs.yml [hunk 14]
```diff
@@ -83,6 +83,8 @@ nav:
               - Start services: cdk/get-started/deploy-rollup/start-services.md
         - Connect to CDK testnet: cdk/get-started/connect-testnet.md
       - How to:
+        - Migrate:
+          - Fork 7 to 8: cdk/how-to/migrate/7-8.md
```
Comment (kmurphypolygon) [inline mkdocs.yml]:
```suggestion
          - Fork migration: cdk/how-to/migrate/fork.md
```

File: docs/cdk/how-to/migrate/7-8.md [hunk 15]
```diff
@@ -0,0 +1,73 @@
...
+
+    8. Wait until the proof is settled on-chain: 
+
+        `SELECT batch_num FROM state.verified_batch ORDER BY batch_num DESC LIMIT 1;` → X
+
+3. Prepare (**do not apply**) new versions according to the version matrix.
+
+4. Stop all services (node, prover/executor, bridge).
+
+
```
Comment (kmurphypolygon) [inline docs/cdk/how-to/migrate/7-8.md]:
```suggestion
```

**Comment by kmurphypolygon** [comment]
OK @Sharonbc01 DONE. I removed all fork numbers to make it generic. https://github.com/0xPolygon/polygon-docs/pull/326 Please approve and merge whenever you want.

**Review by hsutaiyu — APPROVED** [review]
Approved without modifications.

**Review by hsutaiyu — APPROVED** [review]

**Review by Sharonbc01 — APPROVED** [review]

**Pull request merged**